.buildkite/pipeline.yml

```diff
@@ -0,0 +1,47 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  plugins: &common_plugins
+  - &bash_cache automattic/bash-cache#v1.3.2: ~
+  # Common environment values to use with the `env` key.
```

Comment:
Nit:
```suggestion
```
Feels kind of redundant with the previous comment and with the intent for those keys to be self-documenting? 🤷‍♂️

Also we don't have such extra comment for plugins so why have it for env? 🙃